src/qtui/menus.cc

```diff
@@ -149,7 +158,14 @@ QMenuBar * qtui_build_menubar (QWidget * parent)
         audqt::MenuSep (),
         audqt::MenuCommand ({N_("_New"), "document-new", "Ctrl+T"}, pl_new),
         audqt::MenuCommand ({N_("Ren_ame ..."), "insert-text", "F2"}, pl_rename),
+#ifdef Q_OS_MACOS
+        // Command+W is also the system shortcut for closing windows, and thus too
+        // likely too much of a reflex that can lead to closing playlists by accident
+        // (or getting the "are you sure" dialog unintentionally too often).
+        audqt::MenuCommand ({N_("Remo_ve"), "edit-delete", "Ctrl+Shift+W"}, pl_close),
```

Comment:
If we make this change, it shouldn't be Mac-specific.  I'd like more input from others first, though.  The switch to Ctrl-T / Ctrl-W (imitating web browsers) was intentional back when we started the GTKUI.